dev/lib/product_taxonomy/cli.rb

```diff
@@ -95,5 +95,11 @@ def add_attributes_to_categories(attribute_friendly_ids, category_ids)
     def add_value(name, attribute_friendly_id)
       AddValueCommand.new(options.merge(name:, attribute_friendly_id:)).run
     end
+
+    desc "compare_categories VERSION_FOLDER", "Compare category changes between full_names.yml and categories.txt"
+    option :output_dir, type: :string, default: "exports", desc: "Output directory for CSV file"
```

Comment:
It would be more flexible and idiomatic to accept the full output path of the CSV file (filename included), not just output dir